### PR TITLE
Comments: Remove "Approve and reply" placeholder from the reply textarea

### PR DESCRIPTION
**Pull request by Copons**
Remove the "Approve and reply to..." placeholder from the `CommentDetail` reply textarea on non-approved comments, and replace it with "Reply to...".

| Before | After |
| --- | --- |
| screen shot 2017-10-16 at 16 18 31 | screen shot 2017-10-16 at 16 18 37 |

### CODE REVIEW

File: client/blocks/comment-detail/comment-detail-reply.jsx [hunk 1]
```diff
@@ -57,21 +57,15 @@ export class CommentDetailReply extends Component {
 	};
 
 	getTextareaPlaceholder = () => {
-		const { authorDisplayName, comment: { status }, translate } = this.props;
-
-		if ( 'approved' !== status ) {
```
Comment (rodrigoi) [inline client/blocks/comment-detail/comment-detail-reply.jsx]: removing this `if` and keeping the last return should be enough:

```js
getTextareaPlaceholder = () => {
	const { authorDisplayName, translate } = this.props;

	return authorDisplayName
		? translate( 'Reply to %(commentAuthor)s…', {
				args: { commentAuthor: authorDisplayName },
			} )
		: translate( 'Reply to comment…' );
};
```

Comment (Copons) [reply client/blocks/comment-detail/comment-detail-reply.jsx]: I know, it's just that I'm bugged by how Prettier indents this kind of ternaries, breaking ESLint (on a rule not included in the pre-commit hooks, to be fair, but it still shows up as error in my editor).

Comment (kwight) [reply client/blocks/comment-detail/comment-detail-reply.jsx]: I know what you mean @Copons (dang squigglies!), but dropping the `if` does make a nicer diff.

**Comment by Copons** [comment]
@rodrigoi @kwight I've dropped the early return and placed the ternary back.

**Pull request merged**